src/components/sections/Hero/Hero.tsx

```
@@ -43,52 +37,27 @@ export const Hero = (): React.ReactElement => {
-            <span className={styles.tag}>
-              <Icon name="fa-bolt" size="sm" /> Lightning Fast
-            </span>
-            <span className={styles.tag}>
-              <Icon name="fa-lock" size="sm" /> Privacy First
-            </span>
-            <span className={styles.tag}>
-              <Icon name="fa-wifi-slash" size="sm" /> Offline Ready
-            </span>
+          <div className={styles.trustedBy}>
+            <p className={styles.trustedByLabel}>TRUSTED BY TEAMS AT</p>
+            <ul className={styles.companies} role="list">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<ol>
<ul>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)